docs/contributors.json

```
@@ -17,5 +17,6 @@
   "IanVS",
   "ydcjeff",
   "11joselu",
-  "wheatjs"
-]
\ No newline at end of file
+  "wheatjs",
+  "veritem"
+]
```

This file is autogenerated, no need to change it in the PR
```suggestion
  "wheatjs"
]
```